### PR TITLE
Bump version to 2.4.0

### PR DESCRIPTION
**Pull request by jfrerich**
### Summary
Bump version to v2.4.0

Proposal to bump version following Release Notes:
## Summary
Bump version to v2.4.0

Proposal to bump Feature Version to prep for release with the following Release Notes:

### Enhancements 
* 6c581ee Bump version to 2.4.0
* 0a6a9f5 [GH-506] Correctly link or embed attachments (#517)
* 3dea771 Add docker-compose.yml to spin up Jira Server instance (#516)
* 5c00b4f rename `respondErr` function (#513)
* ab82ff2 Add CODEOWNERS and maintainers (#518)
* 09711c0 MM-19674: use respondXXX in http handlers (#500)
* cbd6876 [MM-17816] Add the ability to transition an issue after running `/jira view` (#454)
* b2fb81f MM-21696: New config setting: Hide issue descriptions and comments (#503)
* d1c74e3 Add Site URL to third-party cookie error message (#478)
* 2b7d074 [GH-145] Add link to ephemeral message after uninstall (#469)
* 4fe282e Bump acorn from 5.7.3 to 5.7.4 in /webapp (#498)
* ef13fce MM-18075 Reuse output of `/jira view` in the output of `/jira cr… (#459)
* ea10033 MM-22063 Adding create ticket action support for workflow. (#480)
* 3254af6 Created/Attached to a Jira issue message should include a refere… (#464)
* 56bca86 [GH-167] Render JIRA webhook comments as quoted messages (#467)
* 970e320 [MM-17816] Reporter is not automatically set for `/jira create`… (#462)
* 4de6a53 When posting a response from the Jira plugin, we get React warni… (#466)
* 78afa14 MM-21303 Workflow plugin integration.  (#450)
* 879250a MM-17818 Adding automatic registration of autolinks for JIRA projects. (#453)
* aa13a08 Add unassign slash command (#460)
* 0df9bce [MM-20744] Use FullScreenModal--compact css class, defined by ma… (#457)
* 3ae146d Fixes MM-18737 : Change wording of save subscription buttons (#458)
* c13ae05 [MM-20280] Display reporter name and avatar in `/jira view` outp… (#451)
* b5b54a1 [MM-21762] Refactor jira issue search (#442)
* f0f63da bump go version from 1.12 to 1.13 (#456)
* 272398d Add more manifest metadata (#455)
* 9fc652e [MM-17903] Validate form for subscription create/edit (#432)
* f359a82 [MM-20281] Show channel name at the top of subscription modal wh… (#443)
* c9bfb0c Remove the ji arg of type Instance from UserStore interface method CountUsers (#448)
* 5d8ca77 Add jest watch command (#445)
* 664eb66 MM-16948 Check if Jira server url actually is a Jira Cloud URL (#437)
* 94b23a7 [MM-21218] Display placeholder text when a channel has no subscr… (#436)

### Fixes
* b7404c9 Drop second maintainer from CODEOWNERS (#528)
* e1c2a96 npm -E audit fix (#521)
* ae5676e Fix test under go 1.14 (#508)
* a466955 Fix dead link (#482)
* 67ad86f Error installing Jira when key is too long (#465)
* b5bf3e6 Generate a proper webhook URL (#474)
* 32b3d6b fix help text test (#446)

[MM-17816]: https://mattermost.atlassian.net/browse/MM-17816

### CODE REVIEW

File: .projections.json [hunk 1]
```diff
@@ -0,0 +1 @@
+/Users/jfrerich/dotfiles/mattermost/.projections.json
```
Comment (hanzei) [inline .projections.json:1]: Unrelated change?

Comment (jfrerich) [reply .projections.json]: removed.  That's actually a handy file that makes opening alternative files easy. The most common example is defining test files as alternative files.  https://github.com/jfrerich/dotfiles/blob/master/mattermost/.projections.json

**Comment by jfrerich** [comment]
@DHaussermann gentle reminder for the review!

**Comment by levb** [comment]
https://github.com/mattermost/mattermost-plugin-jira/pull/546 should be included

**Comment by DHaussermann** [comment]
Apologies. Seems I did not post here last week. End to end testing for Jira 2.4 was completed. We were just waiting on 1 fix. 
One this change (linked b y @levb)  is in master we'll be good.

**Comment by jfrerich** [comment]
@DHaussermann, I just merged #546!

**Comment by DHaussermann** [comment]
Tested and passed
- End to end testing done on master
- Confirmed version bump on this PR
- https://github.com/mattermost/mattermost-plugin-jira/pull/546 tested (and briefly merge tested on Master
LGTM @jfrerich  please proceed.
